### DISCUSSION TITLE
Will you consider adding language Settings

### DISCUSSION THREAD

**Discussion by EverythingGoesWithTheFlow** [kingToolbox/WindTerm#43] [Ideas]
After using the software, I think it's very good. Do you consider adding the language setting function？

**Comment by kingToolbox** [comment]
Do you mean supporting multilingual interface? Of course, but it's not the top priority.

Multi-language was already supported at the time of development, but all the interface strings have not been extracted yet, and the translation of various languages also needs the help of the community. In addition, there are many functions to be added, and the design of the entire interface has not yet stabilized.

At present, I am more concerned about fixing bugs, improving the support of various protocols, as well as x11 of Linux version, X/Y/Z modem, memory compression, search sessions, auto-completion and other functions.

Multilingualism is not expected to be released until the second quarter of this year. For WindTerm, it is relatively simple to use and there are not many English professional terms. This is why I think language is not important. What are your thoughts?

**Comment by kingToolbox** [comment]
The new [WindTerm_2.2.0_Prerelease_4](https://github.com/kingToolbox/WindTerm/releases/tag/2.2-prerelease) version has been released and the multilingual interface has been supported, please download and check it, thank you.

**Reply by Miliik** [reply]
Hi ! i would like to help you in the translation of the tool in french and spanish, how can i contribute ?

**Reply by kingToolbox** [reply]
Thank you very much.

I will upload the translation file to the repository recently, and then you can fork and tranlate. Until then, you can open the `WindTerm_installed_path/global/locales/fr.locale` and `WindTerm_installed_path/global/locales/es.locale` for translation. 

@EvoWebFrance has completed the [French translation](https://github.com/kingToolbox/WindTerm/issues/499#issuecomment-1040541769), you can take a look. After the translation, please package the translated `fr.locale` and `es.locale` files and upload it here.
